src/global/BrowserPage.js

```diff
@@ -1,21 +1,13 @@
-const puppeteer = require("puppeteer");
+import 'puppeteer';
 
-let pageInstance = null;
-
-class BrowserPage {
+const BrowserPage = {
   /**
    * @return {Promise<import("puppeteer").Page>}
    */
-  static async getInstance() {
-    if (!pageInstance) {
-      const browser = await puppeteer.launch({ headless: false });
-      const page = await browser.newPage();
-
-      pageInstance = page;
-    }
-
-    return pageInstance;
+  getInstance: async () => {
```

Comment:
This `getInstance` function should behave like a singleton. Otherwise every time I call it will be instantiated a new browser page.